.travis.yml

```diff
@@ -0,0 +1,5 @@
+script:
+- docker build --target test --tag my-test-image .
+- docker run my-test-image ./todo_app/tests/todo_view_model_test.py
+- docker run my-test-image ./todo_app/tests/test_client.py
```

Comment:
You could just do the whole `todo_app/tests` folder rather than a separate docker run for each file.